common-libs/common/src/services/DidDocumentService/ElemDidDocument.ts

```diff
@@ -1,17 +1,16 @@
 import base64url from 'base64url'
 import { parse } from 'did-resolver'
 
-import { op, func } from './elem-lib'
-
-import KeysService from '../KeysService'
+import { func, op } from './elem-lib'
+import { KeyVault } from './KeyVault'
 
 export default class ElemDidDocument {
```

Comment:
Do I understand it right that this PR is only supposed to refactor things and make code more maintainable, without changing the existing or adding some new behavior?

Reply:
Yes, small change that enables the metamask experiment and makes the code easier to maintain.